.travis/update_checkstyle_thresholds

```diff
@@ -36,6 +36,10 @@ should_skip_build() {
     echo "  ...skipping because this repo is not permitted."
     skip=$TRUE
   fi
+  if [[ "$TRIPLEA_RELEASE" != "true" ]]; then
+    echo "  ...skipping because this is not the release build."
+    skip=$TRUE
+  fi
```

Comment:
This check was added to prevent a race between the Java 8 and Java 9 builds to update the thresholds.  As is, the thresholds will only be updated from the Java 8 build (the same build that produces releases).